### PR TITLE
Set "Sections Index" as default area of default (first) author

### PR DESCRIPTION
**Pull request by twiro**
As described here: https://github.com/symphonycms/symphony-2/issues/2529

@nitriques - am I right in open this PR against the 2.6.X-Branch? Or should I use another branch?

### CODE REVIEW

**Comment by nitriques** [comment]
> am I right in open this PR against the 2.6.X-Branch? 

Yes you are! Just check my comments in #2529 :)

File: symphony/lib/core/class.administration.php [hunk 1]
```diff
@@ -149,20 +149,14 @@ private function __buildPage($page)
...
-                            // If there are sections created, redirect to the first one (sortorder)
-                            redirect(SYMPHONY_URL . "/publish/{$section_handle}/");
-                        } else {
-                            // If there are no sections created, default to the Section page
-                            redirect(SYMPHONY_URL . '/blueprints/sections/');
-                        }
+                        // Redirect to the section index if author is a developer
+                        redirect(SYMPHONY_URL . '/blueprints/sections/');
                     } else {
+	                    // Redirect to the author page if author is not a developer
```
Comment (nitriques) [inline symphony/lib/core/class.administration.php]: Could you fix the whitespace issue here please :)

Comment (twiro) [inline symphony/lib/core/class.administration.php]: Fixed :)

**Comment by nitriques** [comment]
Great, this will be merged for 2.7.0 !

**Comment by nitriques** [comment]
Before I can pull this we need this change this line: https://github.com/symphonycms/symphony-2/blob/master/install/includes/install.sql#L13

And also do the change in the upgrade script ?

**Comment by twiro** [comment]
> _Before I can pull this we need this change this line: https://github.com/symphonycms/symphony-2/blob/master/install/includes/install.sql#L13_

> _And also do the change in the upgrade script ?_

Mmh... what would you change it to? We don't have a new default value for **all** types of authors and I don't see a problem keeping `null`.

Apart from setting `/blueprints/sections/`as default area for the **first developer at installation** my PR only changes the redirection after login for authors that actually do have `null` as default area - dependent on whether they are developers or not.

So having `null` in the database still is ok - which also means we wouldn't have to run any database-updates.

**Comment by nitriques** [comment]
Well, ok then!

**Comment by nitriques** [comment]
Sorry I had to delete the merge commit since it's in the wrong branch. It's now 17d87a5

**Pull request merged**